src/test/regress/expected/multi_outer_join.out

```diff
@@ -877,15 +885,15 @@ SELECT create_distributed_table('left_values', 'val');
 
 (1 row)
 
-\copy left_values from stdin
+COPY left_values from stdin;
```

Comment:
Not important if that is just copy&paste error, but is there a reason we needed to change this?

Reply:
No reason. This was a regex error. I can revert this change.

I replaced all `\copy` commands into `COPY` because only the latter support variable interpolation. However, when we do not have reference to local files, both can work just fine.